docs/db.md

```diff
@@ -0,0 +1,45 @@
+# Data Model
+
+## Tables
+
+### Users
+
+| Property | Type    | Notes                                                          |
+| -------- | ------- | -------------------------------------------------------------- |
+| id       | Integer | Auto-generated Primary Key for this row                        |
+| api_id   | String  | 26-character public-facing id made up of numbers and letters   |
+| email    | String  | Email address used to sign up for the service                  |
```

Comment:
nit - for types let's use Postgres types so for String let's declare how many characters we plan to use for the varchar.